### PR TITLE
[WIP] Design tokens

### PR DESCRIPTION
**Pull request by tomquinonero**
This is a fat PR. It includes design tokens. 
It also includes the documentation with it, so please read it.

Few changes:
 - Usage of CSS custom props (IE11 tested!)
 - Introduction of design tokens
 - Feature: script that import kotti colors into figma!
 - Documentation: 3 pages that are related to tokens
 - `yarn tokens`: generates a token.css file that is included in kotti
 - Themable elements:
   - Navbar
   - Accordion
   - Buttons
   - etc
- Lot of style moved to its component
- KtDropdownButton moved to button
- CSS fixes

I know that this PR is big and I'm sorry about that. These matters were hard to address separately.
I would like to present that PR to whom might feel concerned if some of you doesn't get the token principle.

### CODE REVIEW

**Comment by mvangent** [comment]
Cool stuff @tomquinonero, looking forward to testing it once out of WIP!

File: packages/kotti-user-menu/src/UserMenu.vue [hunk 1]
```diff
@@ -126,3 +111,131 @@ export default {
...
+	}
+}
+.user-info-avatar {
+	z-index: 2;
+	flex-grow: 0;
+}
+.user-info-text {
+	z-index: 2;
+	width: 100%;
+	margin-left: 0.4rem;
```
Comment (mvangent) [inline packages/kotti-user-menu/src/UserMenu.vue:152]: @tomquinonero I still see a lot of spacing values defined in the old way, do you not want to update those as well?

**Comment by mvangent** [comment]
@FlorianWendelborn are you onboard with these PR getting merged soon? Can you help us out with the failing pipeline?

File: packages/kotti-user-menu/src/UserMenu.vue (hunk 1 above)
Comment (tomquinonero) [reply packages/kotti-user-menu/src/UserMenu.vue:152]: @mvangent Yes, ideally, but I tried to keep the scope tight on this one!

**Comment by tomquinonero** [comment]
> Hi Tom, I reviewed. It looks good to me. I have one question about refactoring spacing values. often paddings and margins are still hardcoded and not replaced by the unit constants, what is your plan for those?

I am not sure and I hesitate beetwen create `unit-1`, `unit-2` but in CSS or might be that the `rem` value works good enough. We can still change font-size on root to affect them.

**Pull request merged**